### PR TITLE
Add horizontal grid for Slovakia - JTSK03 to JTSK (EPSG:8364)

### LINKED ISSUE #15: Add grids for Slovakia
I have received a confirmation from Slovak geodetic authorities that the grids are now licensed under the terms of CC-BY-ND 4.0. Here is the link for downloads:
https://www.geoportal.sk/en/geodeticke-zaklady/download/

Available grids:
- jtsk03_to_jtsk (horizontal)
- etrs89h_to_evrf2007 (vertical)
- etrs89h_to_baltic1957 (vertical)

From the description of CC-BY-ND license I understand that "NoDerivative" clause does not prohibit us from doing the conversion to GeoTIFF. It would be good if anyone can confirm -

### PR DESCRIPTION
**Pull request by wonder-sk**
Fixes #15 

I have generated the TIF using this command:
```
docker run --rm -v /media:/media osgeo/gdal:alpine-normal-latest python3 $PWD/ntv2_to_gtiff.py  \
    --area-of-use "Slovakia" \
    --copyright "Derived from work by Slovak Geodetic and Cartographic Institute. Creative Commons Attribution 4.0: http://creativecommons.org/licenses/by/4.0/" \
    --source-crs EPSG:8351 \
    --target-crs EPSG:4156 \
    $PWD/Slovakia_JTSK03_to_JTSK.gsb $PWD/sk_gku_JTSK03_to_JTSK.tif
```

I was following the guide at https://github.com/OSGeo/PROJ-data/blob/master/CONTRIBUTING.md - but I got stuck on step 8 - the script was failing in `osgeo/gdal:alpine-normal-latest` container (seems due to the lack of git) and on my base system too (still on gdal 2.x). I will try to get latest GDAL python bindings working on my system (or if anyone would to run that script after this is merged...?)

### CODE REVIEW

**Comment by rouault** [comment]
sk_gku_README.txt missing in travis/expected_main.lst. See https://travis-ci.com/github/OSGeo/PROJ-data/builds/154505234

> but I got stuck on step 8 - the script was failing in `osgeo/gdal:alpine-normal-latest` container

Yeah, I didn't try with it. I run the script locally with my dev env. I can do that step myself after the PR is merged. That's the easy part.

File: copyright_and_licenses.csv [hunk 1]
```diff
@@ -169,6 +169,7 @@ pt_dgt_README.txt,Disclaimed,Public domain
 README.DATA,Disclaimed,Public domain
 se_lantmateriet_README.txt,Disclaimed,Public domain
 se_lantmateriet_SWEN17_RH2000.tif,Lantmäteriet,CC0-1.0
+sk_gku_JTSK03_to_JTSK.tif,GKU Slovakia,CC-BY-4.0
```
Comment (rouault) [inline copyright_and_licenses.csv]:
```suggestion
sk_gku_JTSK03_to_JTSK.tif,Copyright 2013 - GKU Slovakia,CC-BY-4.0
```

**Comment by wonder-sk** [comment]
Thanks for the review, it should be fixed now.

**Comment by rouault** [comment]
Next step for you: a pull request to OSGeo/PROJ to register the grid in grid_alternatives table.
I'm going to refresh the index and update the grid on the CDN

**Pull request merged**